src/network/clientpackethandler.cpp

```diff
@@ -120,7 +121,12 @@ void Client::handleCommand_AuthAccept(NetworkPacket* pkt)
 					<< m_recommended_send_interval<<std::endl;
 
 	// Reply to server
-	NetworkPacket resp_pkt(TOSERVER_INIT2, 0);
+	std::string lang = gettext("LANG_CODE");
```

Comment:
:) NetworkPacket& NetworkPacket::operator<<(const std::string &src) take care of length (16b length as a maximum, but we should raise it in 0.5 to have less problems in the future